Analytics-CSharp/Segment/Analytics/Analytics.cs

```diff
@@ -206,34 +193,34 @@ public void Reset()
 
         private void Startup(HTTPClient httpClient = null)
         {
-            Add(new StartupQueue());
-            Add(new ContextPlugin());
-            Add(new UserInfoPlugin());
+            _ = this.Add(new StartupQueue());
+            _ = this.Add(new ContextPlugin());
+            _ = this.Add(new UserInfoPlugin());
```

Comment:
we probably should just ignore the return values. this looks bad imo

Reply:
Nuked